pulpcore/tests/functional/api/test_correlation_id.py

```diff
@@ -23,12 +23,12 @@ def setUpClass(cls):
         )
         cls.cli_client = cli.Client(cls.cfg)
 
-        cls.orphan_api = OrphansApi(cls.client)
+        cls.orphan_cleanup_api = OrphansCleanupApi(cls.client)
         cls.task_api = TasksApi(cls.client)
 
     def test_correlation_id(self):
         """Test that a correlation can be passed as a header and logged."""
-        response, status, headers = self.orphan_api.delete_with_http_info()
+        response, status, headers = self.orphan_cleanup_api.cleanup_with_http_info({})
```

Comment:
We don't use `status` and `headers` so we should make this
```suggestion
        response = self.orphan_cleanup_api.cleanup({})
```